README.md

```diff
@@ -1,66 +1,78 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Space Travelers Hub
 
-## Available Scripts
+Space Travelers Hub is web application for a company that provides commercial and scientific space travel services. Consists of Rockets, Missions, and the My Profile section. 
 
-In the project directory, you can run:
+> In this application users can book rockets and join selected space missions available to go in the space.
 
-### `npm start`
+## Features
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+- Reserve a Rocket by clicking `Reserve Rocket` button.
+- Cancel Reservation of a Rocket.
+- Join a Mission by clicking `Join mission` button in the Missions' tab.
+- Leave a Joined mission.
+- See reserved rockets and joined mission check `My Profile` tab.
 
+Live Demo [Space Travelers Hub](https://space-traveler-hubs.herokuapp.com/)
+=======
 ### `npm test`
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Built With
 
-### `npm run build`
+- React
+- Redux
+- CSS3
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+## Clone the project
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+You can download or clone this project by running this command from your terminal:
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+git clone https://github.com/Ghilain/space-travelers-hub.git
+This will create a directory in the name of the project folder.
 
-### `npm run eject`
+Once you have the project files and folders, open it with your text editor.
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+Next, switch inside the project directory and run:
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+```
```

Comment:
- [ ] **Required:** To have a professional setup README for a project that has tests, kindly add the **Testing** section with the instructions on how your code should be run.